gulpfile.js

```diff
@@ -1298,6 +1564,58 @@ export async function test() {
     { pattern: "Specs/TestWorkers/**", included: false },
   ];
 
+  let proxies;
+  if (workspace) {
+    // Setup files and proxies for the engine package first, since it is the lowest level dependency.
+
+    files = [
+      {
+        pattern: `packages/${workspace}/Build/Specs/karma-main.js`,
+        included: true,
+        type: "module",
+      },
+      {
+        pattern: `packages/${workspace}/Build/Specs/SpecList.js`,
+        included: true,
+        type: "module",
+      },
+      { pattern: "Specs/Data/**", included: false },
+      { pattern: "packages/engine/Build/Workers/**", included: false },
+      { pattern: "packages/engine/Source/Assets/**", included: false },
+      { pattern: "packages/engine/Source/ThirdParty/**", included: false },
+      { pattern: "packages/engine/Source/Widget/*.css", included: false },
+      { pattern: "Specs/TestWorkers/**/*.wasm", included: false },
+      { pattern: "Specs/TestWorkers/**", included: false },
+    ];
+
+    proxies = {
+      "/base/Build/CesiumUnminified/Assets/":
+        "/base/packages/engine/Source/Assets/",
+      "/base/Build/CesiumUnminified/ThirdParty/":
+        "/base/packages/engine/Source/ThirdParty/",
+      "/base/Build/CesiumUnminified/Widgets/CesiumWidget/":
+        "/base/packages/engine/Source/Widget/",
+      "/base/Build/CesiumUnminified/Workers/":
+        "/base/packages/engine/Build/Workers/",
+    };
+
+    if (workspace === "widgets") {
```

Comment:
@sanjeetsuhag I don't believe the images or css actually get used in the widget unit tests and that this block can safely be omitted.